doc/subsystems/shell/shell.rst

```diff
@@ -365,11 +365,19 @@ The shell module supports the following meta keys:
      - Action
    * - ctrl + a
      - Moves the cursor to the beginning of the line.
+   * - ctrl + b
```

Comment:
You might want to change all these to use ``:kbd:`ctrl + b` `` for a prettier display (as you used at the top of this document).